netlib/http/status_codes.py

```diff
@@ -104,3 +106,5 @@
     INSUFFICIENT_STORAGE_SPACE: "Insufficient Storage Space",
     NOT_EXTENDED: "Not Extended"
 }
+
+RESPONSES = {k: v.encode() for k, v in six.iteritems(RESPONSES)}
```

Comment:
Similar situation here: we may want to `.encode()` in pathod - the fewer we have to deal with bytes, the happier we'll be.